source/routing/raptor_api.cpp

```diff
@@ -509,31 +544,78 @@ static void add_isochrone_response(RAPTOR& raptor,
     }
 }
 
-static std::vector<std::pair<type::idx_t, navitia::time_duration> >
-get_stop_points( const type::EntryPoint& ep,
-                 const type::Data& data,
-                 georef::StreetNetwork& worker,
-                 bool use_second = false) {
+template <typename T>
+const std::vector<georef::Admin*>& get_admins(const std::string& uri, const std::unordered_map<std::string, T*>& obj_map) {
+    auto it = obj_map.find(uri);
+    if (it == obj_map.end()) {
+        static const std::vector<georef::Admin*> empty_list;
+        return empty_list;
+    }
+    return it->second->admin_list;
+}
+
+const std::vector<georef::Admin*> find_admins(const type::EntryPoint& ep, const type::Data& data) {
```

Comment:
const is useless or you forgot a reference. If you opt for the reference, be careful, returning a new object will be buggy